arch/arm/boot/dts/Hawaii.dts

```diff
@@ -0,0 +1,210 @@
+/dts-v1/;
```

Comment:
Both CONFIG_ARCH_KONA https://android.googlesource.com/kernel/bcm/+/android-bcm-tetra-3.10-lollipop-mr1-wear-release/arch/arm/configs/tetra_defconfig#36 and CONFIG_ARCH_JAVA https://android.googlesource.com/kernel/bcm/+/android-bcm-tetra-3.10-lollipop-mr1-wear-release/arch/arm/configs/tetra_defconfig#45 are set in tetra_defconfig so I believe those two archs are needed. Anyway, I'm carefully reading all of your comments and this is really great, thanks for your time and help.